src/plugins/intel_gpu/tests/unit/passes/prepare_primitive_fusing_test.cpp

```diff
@@ -777,5 +777,4 @@ TEST(prepare_primitive_fusing, fuse_in_direction_with_lower_processing_number) {
     program_wrapper::apply_opt_pass<prepare_primitive_fusing>(*program, lo);
 
     ASSERT_FALSE(has_node(*program, "eltw5"));
-    ASSERT_EQ(program->get_node("conv1").get_fused_primitives().size(), 5);
```

Comment:
Why do you remove this checking code? I think it is necessary to check  how many  prims are fused.

Reply:
First of all, this test This is because dGPU has different fusion result. If this assertion needs to remain, it should be limited to iGPU.